### ISSUE TITLE
CentOS - the role is not correcty applied when libselinux-python is not installed

### ISSUE THREAD

**Issue by it-praktyk** [reallyenglish/ansible-role-ntpd#8]
For situation when libselinux-python doesn't exist on a managed node error like below is displayed.

```
TASK [reallyenglish.ntpd : Create ntpd.conf] ***********************************
fatal: [labor-2]: FAILED! => {"changed": true, "failed": true, "msg": "Aborting, target uses selinux but python bindings (libselinux-python) aren't installed!"}

NO MORE HOSTS LEFT *************************************************************
```

I think that verifying if the libselinux-python package is installed should be prerequisites for ntpd role.

Thank you for your work!

**Comment by trombik** [comment]
Thank you for finding a bug. The bug was overlooked because the image used in testing has the package installed by default. Out of curiosity, which image are you using? I am asking because my installation script does not explicitly install the package. Is it wrong to assume that the package is installed by default? If it is, I need to fix the bug in almost every package that supports CentOS.

Here is the file used to create the image. https://github.com/reallyenglish/packer-templates/blob/reallyenglish-master/centos-7.2-x86_64.json

**Comment by it-praktyk** [comment]
I tested the role using the official Centos/7 image for Vagrant https://atlas.hashicorp.com/centos/boxes/7; a vmware_workstation provider.

**Comment by trombik** [comment]
released as 1.1.2, thanks!
